### DISCUSSION TITLE
Seq in a high redundant environment

### DISCUSSION THREAD

**Discussion by feitzi** [datalust/seq-tickets#1203] [Help]
We are highly interested to move our logging infrastructure to seq. 
Our datacenter is geo-redundant, and therefore the centralized logging server needs to support clustering. 

Is this possible with Seq? As far as we know seq runs just on a single node and doesn't support running on multiple hosts? How do you deal with this use case?

**Comment by nblumhardt** [comment]
Hi Mathias,

Thanks for getting in touch!

We're really keen to support this use-case; we're just now in the process of putting the finishing touches on a public preview of "step 1", which is supporting redundant deployment of two Seq nodes and manual fail-over between them: https://github.com/datalust/seq-tickets/issues/1102.

Following on from that, we're aiming to support 3+ node deployments and high-availability next. It's hard to put a date on it, but our goal is to have preview-level support this year.

How does this line up with your time frames? If you're keen to dig deeper into it, we'd be happy to run through it with you on a call - just let us know via `support@datalust.co` if so, and we'll set something up.

Best regards,
Nick

**Reply by CrossBound** [reply]
FYI, I'd be happy to be involved in any kind of beta if that is helpful.

**Reply by nblumhardt** [reply]
That's awesome, thanks @CrossBound. We should have some more public docs up soon, but I've sent an email with some information regarding where this is at, and what the solution looks like right now.

**Comment by CrossBound** [comment]
I am also interested in this. We have a primary datacenter and a secondary datacenter. I would like the ability to support an active-active logging infrastructure between them.
